DatadogCore/Tests/Datadog/Core/_InternalProxyTests.swift

```diff
@@ -12,8 +12,10 @@ import TestUtilities
 class _InternalProxyTests: XCTestCase {
     func testWhenTelemetryIsSentThroughProxy_thenItForwardsToDDTelemetry() throws {
```

Comment:
question/ What is the difference between tests in this file and the ones we change in 
* [DatadogCore/Tests/Datadog/InternalTelemetryTests.swift](https://github.com/DataDog/dd-sdk-ios/pull/1421/files#diff-e07ddd9caba826b1467dd5f6a90924c5f054378cee4795832284db63b33e8f95)?

I think they are so similar that we can merge them into single file. Both work with mocks and seem to assert the same things.

Reply:
Indeed 👍